public/web-app/cypress/e2e/connecte-tests.cy.js

```diff
@@ -0,0 +1,13 @@
+// test connecte client
+describe('template spec', () => {
```

Comment:
Ton describe et ton it doivent être plus parlants
Par exemple :
describe('Tests de connexion de l’utilisateur', () => {
  it('devrait permettre à un utilisateur de se connecter via le menu', () => {
  
  Renommer aussi connecte-tests.cy.js